mesh/mesh.go

```diff
@@ -24,7 +24,7 @@ const (
 	layerSize = 200
 
 	// Genesis layer index
-	Genesis = types.LayerID(0)
+	Genesis = types.LayerID(5)
```

Comment:
this is used mostly in tortoise so I wonder how this change affects it.

Reply:
I need to re visist this but I think it is not used at all already